pkg/transformer/openshift/openshift.go

```diff
@@ -135,6 +135,24 @@ func getAbsBuildContext(context string, composeFileDir string) (string, error) {
 		return "", err
 	}
 	prefix := strings.Trim(string(out), "\n")
+	index := strings.Index(context, prefix)
+	if index != -1 {
+		if prefix == "" {
+			composeFilesplit := strings.Split(composeFileDir, "/")
+			contextsplit := strings.Split(context, "/")
+			var i int
+			for i = range contextsplit {
+				if contextsplit[i] == composeFilesplit[len(composeFilesplit)-1] {
+					break
```

Comment:
if element in contextsplit is equal to last element of composefilesplit, then stop the loop, thats why I put `break` there